### PR TITLE
Feature/natural areas

### PR DESCRIPTION
**Pull request by weiqi-tori**
## Did you read the Contributor Guide?

- Yes, I have read [How to contribute](https://github.com/wri/cities-cif#How-to-contribute)

- No, I haven't read it.

## Is this PR related to a JIRA ticket?

- Yes, the URL of the associated JIRA ticket is https://gfw.atlassian.net/browse/CIF-XXX. The PR name follows the format `[CIF-XXX] my subject`.

- No, this is a documentation update. The PR name follows the format `[DOCS] my subject`.

## Did this PR include necessary documentation updates?

- Yes, I am adding a new API feature.
- Yes, I have updated the documentation update.
- No, this PR does not affect any public API so no need to change the docs.

### CODE REVIEW

File: cities_indicators/indicators/non_tree_cover_by_land_use_gee.py [hunk 1]
```diff
@@ -0,0 +1,178 @@
+from ..layers.non_tropical_tree_cover import NonTropicalTreeCover
+from ..layers.land_cover_class import LandCoverClass
+import ee
+import geemap
+import pandas as pd
+import math
+
+
+class NonTreeCoverByLandUseGEE:
```
Comment (jterry64) [inline cities_indicators/indicators/non_tree_cover_by_land_use_gee.py:9]: @weiqi-tori I think this there's a couple things in here from your other PR that don't seem relevant to this PR

File: cities_indicators/indicators/natural_areas.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+from ..layers.esa_world_cover import EsaWorldCover
+from ..io import to_raster
+
+from xrspatial import zonal_stats
+import xarray as xr
+import numpy as np
+
+
+class NaturalAreas:
```
Comment (jterry64) [inline cities_indicators/indicators/natural_areas.py:9]: I would actually define this as a subclass of EsaWorldCover where it's just mapped to these values, and then have the indicator do stats on that

**Review by jterry64 — CHANGES_REQUESTED** [review]

File: cities_indicators/indicators/non_tree_cover_by_land_use_gee.py (hunk 1 above)
Comment (weiqi-tori) [reply cities_indicators/indicators/non_tree_cover_by_land_use_gee.py:9]: The [feature/natural_areas](https://github.com/wri/cities-cif/tree/feature/natural_areas) branch is based on the [feature/non_tree_cover_by_land_use](https://github.com/wri/cities-cif/tree/feature/non_tree_cover_by_land_use) branch. Since the previous branch/PR has not been merged, the file changes also showed up here.

File: cities_indicators/indicators/natural_areas.py (hunk 2 above)
Comment (weiqi-tori) [reply cities_indicators/indicators/natural_areas.py:7]: I tried to make the change. Is that what you suggested on the subclass?

File: cities_indicators/indicators/non_tree_cover_by_land_use_gee.py (hunk 1 above)
Comment (chrowe) [reply cities_indicators/indicators/non_tree_cover_by_land_use_gee.py]: I changed the PR to be based on non_tree_cover_by_land_use

**Comment by chrowe** [comment]
@weiqi-tori is this ready to review or should we make it a draft?

**Comment by chrowe** [comment]
Closing because this was made before the refactor and not relevant any more.
